istioctl/cmd/root.go

```diff
@@ -144,6 +146,9 @@ func GetRootCmd(args []string) *cobra.Command {
 		Long: `Istio configuration command line utility for service operators to
 debug and diagnose their Istio mesh.
 `,
+		Run: func(cmd *cobra.Command, args []string) {
+			_ = cmd.Help()
```

Comment:
do we need this?

Reply:
If so should we use RunE and handle the error?

Reply:
Yes, the command should be runnable. Otherwise the options cmd will not be recognized in the templater. Revised to RunE.